src/app.html

```diff
@@ -15,25 +15,32 @@
         <div class="headerConnectButton">
           <connectbutton></connectbutton>
         </div>
-        <div class="mobilemenuButton"><img click.delegate="toggleMobileMenu()" src="/Icon_Menu.svg" /></div>
-      </div>
-      <div class="mobileMenuContainer ${showingMobileMenu ? 'showingMobileMenu' : ''}">
-        <div class="menu">
-          <networkfeedback></networkfeedback>
-          <div class="mobileConnectButton">
-            <connectbutton hide-balances></connectbutton>
-          </div>
-          <div class="menuitems">
-            <header-buttons vertical on-navigate.call="onNavigate()"></header-buttons>
-          </div>
+        <div class="mobilemenuButton ${showingMobileMenu ? 'showingMobileMenu' : ''}" click.delegate="toggleMobileMenu()">
```

Comment:
whaaa?  Why is this needed?  If absolutely necessary, please add a comment in the code.

Reply:
Here I've added a new hamburger icon element, which is easier to animate into an `X` when the menu is open.